src/server/test/web/readingsBarGroupFlow.js

```diff
@@ -9,23 +9,112 @@
 
 const { chai, mocha, app } = require('../common');
 const Unit = require('../../models/Unit');
-// const { prepareTest,
-//     parseExpectedCsv,
-//     createTimeString,
-//     expectReadingToEqualExpected,
-//     getUnitId,
-//     ETERNITY,
-//     METER_ID,
-//     unitDatakWh,
-//     conversionDatakWh,
-//     meterDatakWh } = require('../../util/readingsUtils');
+const { prepareTest,
+    parseExpectedCsv,
+    createTimeString,
```

Comment:
This import and the last three are not used. Given the current planned test cases, none will be used (certainly not the kWh ones). Could you please remove them.